rainier-core/src/main/scala/com/stripe/rainier/core/Continuous.scala

```diff
@@ -206,3 +217,41 @@ object Uniform {
   def apply(from: Real, to: Real): Continuous =
     standard.scale(to - from).translate(from)
 }
+
+/**
+  * A mixture distribution composed of continuous distributions
+  *
+  * @param pmf A Map with keys representing component beta distributions and values corresponding to the probabilities of those components.
+  * @param _supportTransform A function mapping a real-valued variable to the range of support of the combined mixture.
+  * @param _supportTransformLogJacobian The jacobian (derivative with respect to v) of the _supportTransform function
+  */
+final case class ContinuousMixture(
```

Comment:
Should we just call this `Mixture` and get rid of the other one? All of the near-term applications I can think of for `Mixture` are of `Continuous` dists anyway. And it's confusing to have both of them if we don't need it.

Reply:
One can imagine a mixture of discrete distributions, right? That's why I specified `Continuous`. Agree the existing `Mixture` is confusing.

Reply:
So mixtures of `Categorical` is just another `Categorical`. If you get into mixtures of discrete but unbounded dists, then yes, you'd need something special for that (like a mixture of `Poisson`). But that's quite a special case that I'm not too worried about supporting for the time being.